### ISSUE TITLE
Binaries For Testing

### ISSUE THREAD

**Issue by asmodat** [cosmos/ledger-cosmos#17]
Could binaries for testing new cosmos ledger app be published on github with instructions on how to flash them into the device? This would be a huge help for those trying to support other platforms.

**Comment by jleni** [comment]
yes, starting from v3.0.x, all releases will generate a script that can be used for testing purposes.
At the moment, it will be temporarily disable until we add a warning to the user interface.
But this is going to be enabled and part of the normal release process starting early next week.

**Comment by jleni** [comment]
reopen because this is temporarily disabled until we add a warning

**Comment by jleni** [comment]
Installer is now generated and published by CI
